Sources/SwifterSwift/UIKit/UIViewControllerExtensions.swift

```diff
@@ -9,6 +9,33 @@
 #if canImport(UIKit) && !os(watchOS)
 import UIKit
 
+// MARK: - Helper Protocols
+protocol Storyboarded {
```

Comment:
We have kind of a "rule" against creating new types, protocols, and stuff... to ensure we keep this simple and only quick and helpful extensions that can be copy/pasted. That one of the things that allow this library not to blow out of proportion :)

Reply:
I see. Understood